lib/floe/workflow/runner/docker.rb

```diff
@@ -74,7 +74,13 @@ def success?(runner_context)
 
         def output(runner_context)
           output = docker!("logs", runner_context["container_ref"], :combined_output => true).output
-          runner_context["output"] = output
+
+          marker_location = output.rindex(OUTPUT_MARKER)
+          return if marker_location.nil?
```

Comment:
no marker = bad output. or assume all is output?

I guess if we don't require it, and then a gem gets updated and randomly outputs, it would be sporadic. (legit case you are seeing)

If we do require it, then we'll force this to be present, and that will be more resilient.

Reply:
Yeah I'd prefer we require it, that way we know for sure anything without it is "logs" at runtime